recipe/meta.yaml

```diff
@@ -9,8 +9,13 @@ source:
   sha256: d267c0c7d6fb30c9ea7e029ef27dadb26ffd7b92cb4b4703f9a3ba0e8f393a1b
 
 build:
-  number: 4
+  number: 5
   skip: true  # [win and vc<14]
+  run_exports:
+    - {{ pin_compatible('curl', max_pin='x') }}
+    - {{ pin_compatible('fftw', max_pin='x') }}
+    - {{ pin_compatible('gdal', max_pin='x.x') }}
+    - {{ pin_compatible('zlib', max_pin='x.x') }}
```

Comment:
I know we want to pin gdal to 3.6 because of https://github.com/GenericMappingTools/pygmt/issues/2215, and I kinda want to keep pinning zlib because there seems to be an issue with that before (though I can't find it anymore). Not sure about whether to keep pinning curl or fftw though.